CHANGELOG.md

```diff
@@ -6,6 +6,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased 1.3.z]
 
+## [1.3.2] - 2025-10-02
+
+> Ночь, улица, фонарь, аптека...
+
+### Changed
```

Comment:
Shall we change it to Breaking?

Reply:
I'm not sure I agree it's a breaking change.

Reply:
Same here. I'd not say it's a breaking change, it should be something that almost no one really looks. In fact, the lack of this change is causing issues in prod for users.